src/miner.cpp

```diff
@@ -559,7 +512,10 @@ bool CreateRestOfTheBlock(CBlock &block, CBlockIndex* pindexPrev,
                 LogPrint(BCLog::LogFlags::NOISY,
                          "Not including tx %s  due to TxFees of %" PRId64 ", bare min fee is %" PRId64,
                          tx.GetHash().GetHex(), nTxFees, nMinFee);
-                continue;
+
+                // Since transactions are sorted by fee, the fee of rest must also be lower
+                // than required.
+                break;
```

Comment:
Yes. I agree with this. Once we pulled the priority stuff out, this becomes true.